packages/frontend/app/models/identity.ts

```diff
@@ -8,11 +8,16 @@ export interface PublicKey {
 export default class Identity extends Model implements Partial<PublicKey> {
   @attr() name!: string;
   @attr() publicKey!: Uint8Array;
+  @attr() verified!: boolean;
 
   get publicKeyAsHex() {
     return toHex(this.publicKey);
   }
 
+  get verification() {
+    return this.verified ? 'models.identity.verified' : 'models.identity.unverified';
```

Comment:
what are these strings here?

Reply:
Strings to display the contact verification status in the contacts table. I know you mentioned possibly using a symbol [here](https://github.com/NullVoxPopuli/emberclear/issues/58) but I thought using a string will be simpler in the beginning.

Reply:
as a part of _"Separation of Concerns"_, models (data), should not have display/view concerns. It makes maintenance and testing harder. :(

Reply:
Makes sense, I'll just call the function from the .hbs and act accordingly